http/client.go

```diff
@@ -537,7 +579,19 @@ func (c *InternalClient) ImportRoaring(ctx context.Context, uri *pilosa.URI, ind
 		uri = c.defaultURI
 	}
 
+	// Set up import options.
+	options := &pilosa.ImportOptions{}
+	for _, opt := range opts {
+		err := opt(options)
+		if err != nil {
+			return errors.Wrap(err, "applying option")
+		}
+	}
+
 	url := fmt.Sprintf("%s/index/%s/field/%s/import-roaring/%d?remote=%v", uri, index, field, shard, remote)
+	if options.Clear {
+		url += "&clear=true"
```

Comment:
I think it's better to use `url.Values` to create the query part of the URL instead of string concatenation. Something like:
```go
url := fmt.Sprintf("%s/index/%s/field/%s/import-roaring/%d", uri, index, field, shard)
vs := url.Values{}
vs.Set("remote", remote.String())
if options.Clear{
    vs.Set("clear", "true")
}
url = fmt.Sprintf("%s?%s", url, vs.Encode())
```